### ISSUE TITLE
all_sets

### ISSUE THREAD

**Issue by wwood** [jogru0/disjoint#5]
Hi,

I'm looking here as a replacement for the now defunct partitions.

Small thing first, in the docs it says

```
// Adding 'd', not joined to anything.
dsv.push('c'); // {'a', 'b'}, {'c'}
```

You are pushing c, not d.

But to the real question, is there a way to extract the sets, similar to the `all_sets()` in `partitions`?

**Comment by jogru0** [comment]
Hi!

Thanks for your interest.

Good catch in the docs, I'll fix that.

Regarding your question about extracting sets, does this solve your use case? https://docs.rs/disjoint/latest/disjoint/struct.DisjointSet.html#method.sets

**Comment by jogru0** [comment]
The typo in the docs is fixed with v0.7.0. Thanks for reporting!
